services/web-app/src/lib/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const BASE_URL = "https://mylf3hxjs8.execute-api.eu-west-2.amazonaws.com/prod";
+export const BASE_URL = "https://2dw4kjr0a2.execute-api.eu-west-2.amazonaws.com/prod";
```

Comment:
We need to chat about how to better do this. Not now's problem, but would be nice if this could be gotten from aws or something!

Reply:
This will is solved already. We will use the staging URL here in future. That is deployed.